pkg/oauth/tokenrequest/request_token.go

```diff
@@ -55,12 +59,44 @@ type RequestTokenOptions struct {
 	OsinConfig   *osincli.ClientConfig
 	Issuer       string
 	TokenFlow    bool
+
+	AuthorizationURLHandler AuthorizationURLHandlerFunc
+	LocalCallbackServer     *callbackServer
```

Comment:
To avoid repeating the same info, isn't the description of the entry func enough? https://github.com/openshift/library-go/pull/1531/files#diff-f30259f7a13841a226555f2801678f096da8bb80456cf414201b8cfd647da728R82-R87

Reply:
Added comments: https://github.com/openshift/library-go/pull/1531/commits/450db3ff01b29005d8be93c5459baabca267652b#diff-f30259f7a13841a226555f2801678f096da8bb80456cf414201b8cfd647da728R63-R69